app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from time import sleep
+from datetime import datetime
+
+
+def loging():
+    while True:
+        with open(datetime.now().strftime("app-%H_%M_%S.log"), "w+") as file:
+            file.write(str(datetime.now()))
+        print(str(datetime.now()), datetime.now().strftime("app-%H_%M_%S.log"))
```

Comment:
```suggestion
        print(datetime.now(), datetime.now().strftime("app-%H_%M_%S.log"))
```
`str` is not required inside `print` function